### ISSUE TITLE
A11y_Code.visualstudio.com_MS software license and terms_MS Visual Studio Code_Info & relationships : The page does not have Heading level 1.

### ISSUE THREAD

**Issue by MukallaDivya** [microsoft/vscode-docs#5167]
## GitHub Tags:
#A11yTCS;#Win10;#WCAG1.3.1;#DesktopWeb;#A11ySev2;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;#Element:Heading;#AI4W;#A11yMAS;
## Actual Result :
In this page, There is no H1 heading.
## Expected Result :
Heading level 1 should be provided to every page.
## Environmental Details:
#URL: https://code.visualstudio.com/License
## OS & Browser Details :
Windows 10 Enterprise 21H2(OS Build 19044,1466)
Microsoft Edge Version 98.0.1108.43 (Official build) (64-bit)
## Repro steps:
1. Launch the URL https://code.visualstudio.com/License.
2. Turn on the tool "Accessibility Insights for web"
3. Now, Turn on headings.
4. Observe the page , H1 is provided or not.
## User Impact :
User not able to understand the current structure of the page, if H1 is not provided.
## Attachments :
![Bug_5167_No heading level 1](https://user-images.githubusercontent.com/99255594/153325838-0c01939f-a855-4336-9f6c-644355b177a7.jpeg)

**Comment by gregvanl** [comment]
This text and formatting come from Microsoft CELA so we are not able to change this legal document at all or risk non-compliance.

**Comment by MukallaDivya** [comment]
We have discuused this with our accessibility SME and we agreed to close the bug as per the above justification provided by Gregvanl.

**Comment by rzhao271** [comment]
@MukallaDivya the issue has been fixed on the site.

**Comment by MukallaDivya** [comment]
check again this bug in the below environment.
URL : https://code.visualstudio.com/License.
Now, we are seeing that, one H1 level heading is defined for a page.
So, the issue is fixed.
![Bug_5167_issue is fixed](https://user-images.githubusercontent.com/99255594/157379737-1c279282-d891-42af-af5f-1539c4aa2a7f.png)
